django_datadog_logger/middleware/request_id.py

```diff
@@ -11,22 +11,17 @@ def generate_request_id():
 
 def get_or_create_request_id(request):
     request_id = request.META.get("HTTP_X_REQUEST_ID")
-    if request_id and re.match("^[a-zA-Z0-9+/=-]{20,200}$", request_id):
-        return request_id
-    else:
-        return generate_request_id()
+    return request_id or generate_request_id()
```

Comment:
we can't simply remove code that validates request id to look like a uuid or something similar. there have been cases of garbage text submitted in this header field that causes trouble in logs later on